javatests/arcs/core/storage/ReferenceModeStoreTest.kt

```diff
@@ -771,7 +769,7 @@ class ReferenceModeStoreTest {
 
     // region Helpers
 
-    private fun DirectStoreMuxer<CrdtData, CrdtOperation, Any?>.getEntityDriver(
+    private fun DirectStoreMuxer<CrdtEntity.Data, CrdtEntity.Operation, CrdtEntity>.getEntityDriver(
```

Comment:
At the moment, we could even remove the type params on DSM, making it always generation CrdtEntity stores. 

But maybe down the road we'll want something else, so I guess it doesn't hurt to leave it like this.